packages/component-library-css/rollup.config.js

```diff
@@ -0,0 +1,46 @@
+import typescript from '@rollup/plugin-typescript';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import svgr from '@svgr/rollup';
+import _ from 'lodash';
+import path from 'path';
+import postcss from 'rollup-plugin-postcss';
+
+import tsconfig from '../../tsconfig.json';
+
+const externalDependencies = [
+  '@material-ui/core',
+  '@material-ui/lab',
+  '@material-ui/pickers',
+  'react',
+  'react-dom',
+  '@material-ui/core/transitions',
+  '@material-ui/core/OverridableComponent',
+];
+const internalDependencies = _.keys(tsconfig.compilerOptions.paths);
+const dependencies = externalDependencies.concat(internalDependencies);
```

Comment:
Do we actually need a separate rollup config for this package? In any case, I believe there shouldn't be need for the external code if we're only exporting css from this package.

Reply:
> @bddjong Fixed the broken visual regression tests. Some context:
> 
> Adding `extract: true` to the `postcss` plugin in rollup seems to break some styling for a few components, resulting in the failed visual regression tests. For now I've removed the `extract: true` from our rollup config, duplicated the rollup config to the css package folder and added `extract: true` to that specific config. Once we've worked out why the styling breaks for these specific components and we've re-added the `extract: true` to the global rollup config we can remove the specific one and point to the global file again.
> 
> For now fixing this seems like out of scope for this specific issue though, which is why I went with this approach.

This comment explains the need for the seperate rollup config, probably could indeed use some changes but as it's not intended to be used long-term not sure whether this is worth our time.